.gitignore

```diff
@@ -7,3 +7,6 @@ yarn-error.log
 **/.sync.ffs_db
 .direnv
 .log
+
+.vscode
+todo.txt
```

Comment:
No need to change this, but it would probably make more sense to put this in your [global gitignore file](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c) rather than here

Reply:
I won't touch it for now, but hot tip!